packages/office-ui-fabric-react/src/components/Check/Check.styles.ts

```diff
@@ -0,0 +1,136 @@
+import { ICheckStyleProps, ICheckStyles } from './Check.types';
+import {
+  concatStyleSets,
+  getTheme,
+  HighContrastSelector,
+  IStyle,
+  ITheme,
+} from '@uifabric/styling';
+import { memoizeFunction } from '@uifabric/utilities';
+
+const DEFAULT_CHECKBOX_HEIGHT: string = '18px';
```

Comment:
We shouldn't "hard code" this value unless the consumer will never need to change it. It seems like in this case we should expose this as a public prop.

Reply:
```
const {
    checkBoxHeight = DEFAULT_CHECKBOX_HEIGHT,
    checked,
    className,
    theme,
  } = props;
```

Setting checkBoxHeight to equal that variable is just the way to set it as default and is overridden by what the user set in props. To be honest, I didn't need DEFAULT_CHECKBOX_HEIGHT, but I thought it'd be a nice easy way to see/change the default later down the road.

Reply:
@battletoilet Could you take another look at this review?